web/rootfs/etc/cont-init.d/10-config

```diff
@@ -92,6 +92,13 @@ if [[ ! -f /config/config.js ]]; then
              -e "s#// transcribingEnabled:.*#transcribingEnabled: true,#" \
              /config/config.js
     fi
+
+    if [[ $TURN_ENABLE -eq 1 || "$TURN_ENABLE" == "true" ]]; then
```

Comment:
We only check for 1 for the other ENABLE type settings...

Reply:
not for all. I don't see nothing bad to check `1` and `true`.

Reply:
Conincidentally it's only on the ones you added ;-) Ok then.

Reply:
Yep ;)
I look at it from the point of view variables in configmap of k8s. So, `true` is more comfortable to use there.

Reply:
Yeah, makes sense!